sys/mips/cbus.c

```diff
@@ -16,10 +16,5 @@ static bus_space_t cbus_bus_space = {
   .bs_write_1 = cbus_write_1,
 };
 
-resource_t cbus_uart[1] = {{.r_bus_tag = &cbus_bus_space,
-                            .r_bus_handle = 0,
-                            /* RT_MEMORY since CBUS UART is a device mapped in
-                             * common physical address space of MIPS CPU. */
-                            .r_type = RT_MEMORY,
-                            .r_start = 0,
-                            .r_end = 7}};
+resource_t cbus_uart[1] = {
+  {.r_bus_tag = &cbus_bus_space, .r_bus_handle = 0, .r_start = 0, .r_end = 7}};
```

Comment:
Could you format it?

Reply:
This is the outcome of the `make format` command. Should I format it manually to make it look like it does now?

Reply:
Look for drivers declaration in `/sys/drv`. They use `/* clang-format off */` and `/* clang-format on */` for that purpose.
It's not _very_ important but from my point of view it's really helpful.

Reply:
Done.